site/docs/v1/tech/admin-guide/monitor.adoc

```diff
@@ -134,6 +136,51 @@ You can also enable JMX metrics as outlined in the link:/docs/v1/tech/admin-guid
 
 The default system metrics are also available in a Prometheus compatible form. link:/docs/v1/tech/tutorials/prometheus[This tutorial explains how to set up Prometheus to scrape the FusionAuth metrics].
 
+== OpenTelemetry
+
+FusionAuth is a Java application. You can use the https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry] project to instrument it with no changes to the code. This will help you diagnose performance or other issues.
+
+FusionAuth contains no custom spans, but it can be set up as an exporter. This can provide useful data about the types of requests, database queries, and more.
+
```

Comment:
Maybe just personal preference, but `without changes` sounds clearer to me than `with no changes`, same for `contains no custom spans`.